ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml

```diff
@@ -3059,10 +3066,19 @@
 		<ns1:IsNormative>Normative</ns1:IsNormative>
 		<ns1:Verification>Test Case</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-OXWSCORE_R2400</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.4.24</ns1:Doc_Sect>
+		<ns1:Description>[In t:ItemType Complex Type] [The element "MimeContentUTF8" with type "t:MimeContentUTF8Type (section 2.2.4.29)"] Specifies an instance of the MimeContentUTF8Type complex type that contains the native MIME stream of an object that is represented in UTF-8.&lt;79&gt;</ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Non-testable</ns1:Verification>
+	</ns1:Requirement>
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R2064</ns1:REQ_ID>

```

Comment:
it should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments